src/library/scala/collection/convert/LongAccumulator.scala

```diff
@@ -242,6 +247,32 @@ object LongAccumulator extends collection.SpecificIterableFactory[Long, LongAccu
   override def empty: LongAccumulator = new LongAccumulator
 
   override def newBuilder: mutable.Builder[Long, LongAccumulator] = new LongAccumulator
+
+  class SerializationProxy[A](@transient private val acc: LongAccumulator) extends Serializable {
+    @transient private var result: LongAccumulator = _
+
+    private def writeObject(out: ObjectOutputStream): Unit = {
+      out.defaultWriteObject()
+      val size = acc.longSize
+      out.writeLong(size)
+      val st = acc.stepper
+      while (st.hasStep)
+        out.writeLong(st.nextStep())
+    }
+
+    private def readObject(in: ObjectInputStream): Unit = {
+      in.defaultReadObject()
+      val res = new LongAccumulator()
+      var elems = in.readLong()
+      while (elems > 0) {
+        res += in.readLong()
+        elems -= 1l
```

Comment:
Can you use `1L` or simply `1` instead of `1l`?  I wondered for an instant why you were subtracting 11, and then wondered about a variable called `l1`.

Reply:
Yeah, sorry!!